cli/src/time_util.rs

```diff
@@ -1,97 +1,37 @@
-use chrono::format::StrftimeItems;
-use chrono::DateTime;
-use chrono::FixedOffset;
-use chrono::LocalResult;
-use chrono::TimeZone as _;
-use chrono::Utc;
+use jiff::fmt::strtime;
+use jiff::tz::Offset;
+use jiff::tz::TimeZone;
+use jiff::Zoned;
 use jj_lib::backend::Timestamp;
-use once_cell::sync::Lazy;
-use thiserror::Error;
 
-/// Parsed formatting items which should never contain an error.
-#[derive(Clone, Debug, Eq, PartialEq)]
-pub struct FormattingItems<'a> {
-    items: Vec<chrono::format::Item<'a>>,
+fn datetime_from_timestamp(context: &Timestamp) -> Result<Zoned, jiff::Error> {
+    Ok(
+        jiff::Timestamp::from_millisecond(context.timestamp.0)?.to_zoned(TimeZone::fixed(
+            Offset::constant((context.tz_offset / 60).try_into().unwrap_or_default()),
+        )),
```

Comment:
Use `Offset::from_hours()` and propagate error? The doc says `Offset::constant()` may panic.

BTW, it's probably better to add `timestamp.to_zoned()`.

Reply:
What timezone would that returned `Zoned` have, though? Do we want to make it take a TZ parameter, or just default to UTC, or system?

Reply:
> What timezone would that returned `Zoned` have, though?

The fixed offset as we do in this function.

Reply:
Oh, wait, sorry, you mean `jj_lib::Timezone::to_zoned` would take the tz as a parameter, right? Not sure why I was thinking otherwise.

Reply:
No, I just meant move `datetime_from_timestamp()` to `Timestamp::to_zoned() -> Result<Zoned, ..>`. The `Timestamp` knows the fixed timezone.